parlai/tasks/coco_caption/agents.py

```diff
@@ -93,6 +120,13 @@ def __init__(self, opt, version='2014'):
             self._setup_image_data()
         self.dict_agent = DictionaryAgent(opt)
 
+    @staticmethod
+    def add_cmdline_args(argparser):
+        agent = argparser.add_argument_group('Comment Battle arguments')
+        agent.add_argument('--no_intro', type="bool",
+                           default=True,
+                           help='Include an intro question with each image for readability.')
```

Comment:
let's make it --include-intro, default=False, and include the actual intro text in the help "e.g. for coco_caption, Describe the above picture in a sentence."